src/main/java/org/sonarsource/scanner/maven/bootstrap/ScannerFactory.java

```diff
@@ -22,38 +22,38 @@
 import java.util.Properties;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.execution.RuntimeInformation;
-import org.sonar.runner.api.EmbeddedRunner;
-import org.sonar.runner.api.LogOutput;
+import org.sonarsource.scanner.api.EmbeddedScanner;
+import org.sonarsource.scanner.api.LogOutput;
 
-public class RunnerFactory {
+public class ScannerFactory {
 
   private final LogOutput logOutput;
   private final RuntimeInformation runtimeInformation;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "RuntimeInformation"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)